src/pcapi/routes/saml/educonnect.py

```diff
@@ -26,9 +26,15 @@
 @blueprint.saml_blueprint.route("educonnect/login", methods=["GET"])
 @authenticated_user_required
 def login_educonnect(user: user_models.User) -> Response:
+    redirect_user = request.args.get("redirect", default=True, type=bool)
```

Comment:
```suggestion
    redirect_user = request.args.get("redirect", default=True, type=lambda v: v.lower() == 'true')
```
Sinon ça vaudra toujours `True`
```python
bool('true') # True
bool('false') # True
```

https://stackoverflow.com/questions/65575796/why-does-the-flask-bool-query-parameter-always-evaluate-to-true